test/keyword_type_test.js

```diff
@@ -19,14 +18,16 @@ describe('KeywordType Controller', function() {
     beforeEach('Populate SQLite memory DB with fresh test data', function() {
         return knex.seed.run();
     });
-    describe('GET Keyword Type', function(){
+    describe('GET /api/keywordtypes/', function(){
```

Comment:
Small nitpick here, this word should be singular, as in 'keywordtype', not 'keywordtypes'.